src/deepsparse/server/build_logger.py

```diff
@@ -190,7 +237,32 @@ def _build_custom_logger(logger_arguments: Dict[str, Any]) -> BaseLogger:
     return logger
 
 
-def _get_target_identifier(endpoint_name: str, target_name: str) -> str:
+def _get_target_identifier(
+    *identifier_strings, is_system_logging_identifier=False
+) -> str:
+    if is_system_logging_identifier:
```

Comment:
why bake this all into one function with an optional bool flag that does completely different functionality? seems like we can just point build system loggers directly to `_get_target_identifier_system`